specs/mcpfile-schema-0.1.0.json

```diff
@@ -218,11 +218,11 @@
           "description": "List of template arguments for the prompt."
         },
         "inputSchema": {
-          "$ref": "#/$defs/Schema",
+          "type": "object",
```

Comment:
@nader-ziada could you explain a bit more why switching from the json schema bit here to just `object` fixes the issue?

Reply:
Okay, I like this change for now then - let's go ahead with it. Let's capture a backlog task of updating the schema to be more detailed than "object" in the future, and try to get it to show something closer to a json schema object describing a json schema